Source/WebCore/page/csp/ContentSecurityPolicyDirectiveList.cpp

```diff
@@ -132,7 +132,7 @@ static inline bool checkMediaType(ContentSecurityPolicyMediaListDirective* direc
 {
     if (!directive)
         return true;
-    if (typeAttribute.isEmpty() || typeAttribute.stripWhiteSpace() != type)
+    if (typeAttribute.isEmpty() || typeAttribute.stripLeadingAndTrailingCharacters(deprecatedIsSpaceOrNewline) != type)
```

Comment:
Since the only use here of the result is to compare with another string this could use the `StringView` version for better efficiency. The cleanest way to do that here is probably to change the type of the `typeAttribute` function argument.

Reply:
I left a FIXME for changing the argument type as it's quite a deep call stack.